rpc/clients/typescript/test/ws_client_test.ts

```diff
@@ -84,7 +84,8 @@ describe('WSClient', () => {
 
             const client = new WSClient(`ws://localhost:${SERVER_PORT}`);
             const perPage = 1;
-            const orderInfos = await client.getOrdersAsync(perPage);
+            const getOrdersResponse = await client.getOrdersAsync(perPage);
```

Comment:
Optional: Nit: Since the only part of `getOrdersResponse` that is used in this test is `ordersInfo`, you could also use object destructuring here.

```
const { ordersInfo } = await client.getOrdersAsync(perPage);
```

Reply:
We care about all three here now.